README.md

```diff
@@ -16,11 +82,49 @@ chrome.runtime.sendMessage(
 );
 ```
 
-Make sure you configure [`externally_connectable`](https://developer.chrome.com/extensions/manifest/externally_connectable) in `manifest.json` to accept messages from the sender extension or web page.
+Make sure you configure externally_connectable in your manifest overrides to accept messages from the sender extension or web page.
+
+You can get your EXTENSION_ID from the Chrome store once you've published your extension, or generate your own for development purposes only.
+
+By default, all mapped content is stored in a default context, so if you don't need to switch contexts you can ignore the concept entirely.
```

Comment:
a section called `# Bringing it all together...` to bring the concept together, perhaps using B12 as an example here